integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -219,6 +226,22 @@ private void writeTest3Util(TachyonURI filePath, ClientOptions op, int len) thro
   }
 
   /**
+   * Test writing to a file and specify the location to be localhost.
+   *
+   * @throws IOException if file can not be opened successfully.
+   */
+  @Test
+  public void writeSpecifyLocalTest() throws IOException, TException {
+    TachyonURI filePath = new TachyonURI(PathUtils.uniqPath());
+    final int LENGTH = 2;
```

Comment:
Right, but it needs to be `static final`? https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names